CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 ## Changed
 - *[#110](https://github.com/idealista/prom2teams/pull/110) Using Alpine Linux instead of Debian Slim in Docker Image* @dortegau
 
+## [2.3.1](https://github.com/idealista/prom2teams/tree/2.3.1)
+[Full Changelog](https://github.com/idealista/prom2teams/compare/2.3.0...2.3.1)
+## Changed
+- *[#112](https://github.com/idealista/prom2teams/issues/112) Allow custom labels in jinja template*
```

Comment:
Could you add your name to that changes?

Reply:
Done